test/parallel/test-util-inherits.js

```diff
@@ -85,7 +85,8 @@ assert.throws(function() {
 }, common.expectsError({
     code: 'ERR_INVALID_ARG_TYPE',
     type: TypeError,
-    message: 'The "superCtor.prototype" argument must be of type function'
+    message: 'The "superConstructor.prototype" argument must be of type \
```

Comment:
Nit: I'd prefer to avoid multiline strings like this. perhaps just:

```js
  message: 
    'The "superConstructor.prototype" argument must be of type function'
```